start-a-local-cluster-in-docker.md

```diff
@@ -1,57 +1,118 @@
+        $("#linuxinstall").hide(); 
+    }
+});
+</script>
+
+<div id="os-tabs" class="clearfix">
+    <button id="mac" class="current">Mac</button>
+    <button id="linux">Linux</button>
+    <button id="windows">Windows</button>
+</div>
 
 Once you've [installed the official CockroachDB Docker image](install-cockroachdb.html), it's simple to run a multi-node cluster across multiple Docker containers on a single host, using Docker volumes to persist node data.
 
-{{site.data.alerts.callout_info}}Running multiple nodes on a single host is useful for testing out CockroachDB, but it's not recommended for production deployments. To run a physically distributed cluster in production, see <a href="manual-deployment.html">Manual Deployment</a>, <a href="cloud-deployment.html">Cloud Deployment</a>, or <a href="orchestration.html">Orchestration</a>.{{site.data.alerts.end}}
+{{site.data.alerts.callout_danger}}Running a stateful application like CockroachDB in Docker is more complex and error-prone than most uses of Docker and is not recommended for production deployments. To run a physically distributed cluster in containers, use an orchestration tool like Kubernetes or Docker Swarm. See <a href="orchestration.html">Orchestration</a> for more details.{{site.data.alerts.end}}
 
-<div id="toc"></div>
+<!--<div id="toc"></div>-->
 
-## Watch a Demo
+<div id="macinstall" markdown="1">
+{% include start_in_docker/mac-linux-steps.md %}
+</div>
 
-Feel free to watch this process in action before going through the steps yourself. Note that you can copy commands directly from the video, and you can use **<** and **>** to go back and forward.
-
-<asciinema-player class="asciinema-demo" src="asciicasts/start-a-local-cluster-docker.json" cols="107" speed="2" theme="monokai" poster="npt:0:43" title="Start a Local Cluster in Docker"></asciinema-player>
+<div id="linuxinstall" markdown="1">
+{% include start_in_docker/mac-linux-steps.md %}
+</div>
 
+<div id="windowsinstall" markdown="1">
 ## Step 1. Create a bridge network
 
 Since you'll be running multiple Docker containers on a single host, with one CockroachDB node per container, you need to create what Docker refers to as a [bridge network](https://docs.docker.com/engine/userguide/networking/#/a-bridge-network). The bridge network will enable the containers to communicate as a single cluster while keeping them isolated from external networks. 
 
-~~~ shell
-$ docker network create -d bridge roachnet
+~~~ powershell
+PS C:\Users\username> docker network create -d bridge roachnet
 ~~~
 
 We've used `roachnet` as the network name here and in subsequent steps, but feel free to give your network any name you like.
 
 ## Step 2. Start your first container/node
 
-~~~ shell
-$ docker run -d \
+{{site.data.alerts.callout_info}}Be sure to replace <code>&#60;username&#62;</code> in the <code>-v</code> flag with your actual username.{{site.data.alerts.end}}
+
+~~~ powershell
+PS C:\Users\username> docker run -d \
```

Comment:
So I just tested this, in PS, you need to use backticks ` instead of backslashes \ to continue commands on the next line.